package.json

```diff
@@ -26,6 +26,7 @@
     "prepare": "husky install"
   },
   "resolutions": {
+    "@types/react": "^17.0.52",
```

Comment:
Is this still necessary?

Reply:
Is this because there could be other versions coming as transitive dependencies?

Reply:
Yes, without this we get a mismatch with `react-google-recaptcha` which uses types v18